.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: required
 branches:
     only:
         - master
+        - bug-1317316
```

Comment:
Please remove this added line.  TravisCI will test the PR without it, with results at https://travis-ci.org/mozilla/kuma/pull_requests